content/en/account_management/billing/profiler.md

```diff
@@ -0,0 +1,110 @@
+| Total          |            |                                                                                                 | $60 + $20      | **$80 per month**    |
+
+
+### Case 4: Hosts with 5 Containers but only 2 are Profiled
+
+Using 5 hosts with 5 containers per each host but only two of them are profiled containers . A profiled container is a container that is running the Continuous Profiler service. This does not include containers that are not being profiled – i.e sending profiling data from the container to the Datadog Agent.
+
+| Billable Unit  | Quantity   | Price                                                                                           | Formula       | Subtotal              |
+|----------------|------------|-------------------------------------------------------------------------------------------------|---------------|-----------------------|
+| APM Hosts      | 5          | $12 per host                                                                                    | 5 * $12       | $60                   |
+| Profiled containers  | 2 per host | $0 since 2 containers is below the allotment limit, no overage charge is issued. The remaining 3 containers not reporting profile data are not counted towards the allotment.         | 0 * $2        | $0                    |
+| Total          |            |                                                                                                 | $60 + $0      | **$60 per month**    |
+
+
+### Case 5: Hosts with Varying Numbers of Profiled Containers
+
+Using 2 profiled hosts - Host A and Host B.
+
+* Host A is running 8 containers
+* Host B is running 2 containers         
+
+All 10 containers are running application instances that are being profiled, meaning Host A has an overage of 4 containers and Host B can add 2 more containers.
+
+In this scenario, we aggregate all containers across all hosts (i.e. 2 hosts, 10 containers). Then we would have 8 containers included in price and an overage of 2 containers. Therefore, the breakdown would look as such:
+
+
+| Billable Unit  | Quantity   | Price                                                                                           | Formula       | Subtotal              |
+|----------------|------------|-------------------------------------------------------------------------------------------------|---------------|-----------------------|
+| APM Hosts      | 5          | $12 per host                                                                                    | 2 * $12       | $24                   |
+| Profiled containers  | 10 in aggregate of host A + host B | $2 per overage container. In this case 2 hosts would allow up to 8 containers but we have 2 containers summed across two hosts: 10-8 = 2 overage containers        | 2 * $2 * 2 hosts        | $8                    |
```

Comment:
```suggestion
| Profiled containers  | 10 in aggregate of host A + host B | $2 per overage container. In this case 2 hosts would allow up to 8 containers but we have 2 containers summed across two hosts: 10-8 = 2 overage containers        | $2 * 2 hosts        | $4                    |
```